theseus_gui/src/components/SideNav.vue

```diff
@@ -0,0 +1,22 @@
+<template>
```

Comment:
does this need to be it's own component? also feels like it could just be in the main page file